data_policy.md

```diff
@@ -46,8 +50,6 @@
 
 3.1.1 All raw data and associated metadata obtained as a result of free (non-commercial) access to ISIS, will be treated as described below, with ISIS acting as the custodian.
 
-3.1.2 All raw data, facility generated reduced data and associated metadata obtained as a result of ‘commercial-in-confidence’ access to ISIS will be owned exclusively by the commercial user.  Commercial users must agree with their relevant instruments scientists how they wish their raw data, facility generated reduced data and metadata to be managed before the start of any experiment.
```

Comment:
I agree it is preferable to delete 3.1.2 and replace it with a sentence in the introduction. But we should check with our industrial person Graham Appleby, including whether "Commercial users must agree with their relevant instruments scientists how they wish their raw data, facility generated reduced data and metadata to be managed before the start of any experiment" is obsolete or other. I will email Graham CC you in

Reply:
Had meeting with Graham and Christ Frost